Add EAP build stage descriptions (#5515)
JetBrains/kotlin-web-site: docs/topics/eap.md

You can participate in the Kotlin Early Access Preview (EAP) to try out the latest Kotlin features before they are released.

> Before every language (_2.x.0_) and tooling (_2.x.20_) release, we ship Early Access Preview (EAP) builds to test in real projects and gather early feedback.

Reviewer: "We ship to test and gather feedback" seems ambiguous: it's the users who test it, and we gather feedback, right? "...to gather early feedback and let you test them in real projects"?

Author: You're right. How about "we ship Early Access Preview (EAP) builds for you to test in real projects and share early feedback"?

Kotlin EAP builds typically include the following stages:

| EAP build | Description                                                                                                                                                                                          |
|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **Beta1** | Introduces the first set of upcoming features, improvements, and other significant changes. Gives you an early opportunity to evaluate new functionality and share feedback.                         |

> | **Beta2** | Usually adds more features and refinements based on feedback we've received. Is feature-complete and continues the preview of the upcoming release, further maturing previously introduced features. |

Reviewer: ```suggestion
| **Beta2** | Usually adds more features and refinements based on the feedback we've received. Is feature-complete and continues the preview of the upcoming release, further maturing previously introduced features. |
```

> | **RC**    | The first release candidate (RC). Focuses on stabilizing the changes delivered in Beta1 and Beta2, as well as fixing regressions found during testing.                                               |

Reviewer: ```suggestion
| **RC**    | The first release candidate. Focuses on stabilizing the changes delivered in Beta1 and Beta2, as well as fixing regressions found during testing.                                               |
```

To me it feels like we introduce the abbreviation in the beginning of the line, but that's up to you.

| **RC2**   | Includes important fixes to finalize the release and confirm readiness.                                                                                                                              | 

> We'd appreciate it if you find and report bugs to our issue tracker, [YouTrack](https://kotl.in/issue). 

Reviewer: ```suggestion
We'd appreciate it if you report found bugs to our issue tracker, [YouTrack](https://kotl.in/issue). 
```

I just think this is neater :)

Author: Hmm this doesn't sound quite natural to me. How about "report any bugs you find"?

Reviewer: Yep, that's better 👍 
I just thought asking to look for bugs can seem a bit presumptuous.
